src/frontend/components/Button.spec.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { matches } from './test-utils';
+import Button from './Button';
+
+describe('Button', () => {
+  it('matches snapshot', () => {
+    matches(<Button>Sign in</Button>);
```

Comment:
Remove the sign in from the test. otherwise the test will fail if it doesnt see 'sign in'